source/crud/crud.rst

```diff
@@ -1129,148 +1237,183 @@ Find And Modify
   class FindOneAndDeleteOptions {
     
     /**
+     * Specifies a collation.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.4, the driver MUST raise an error if the caller explicitly provides a value.
+     *
+     * @see https://docs.mongodb.org/manual/reference/command/findAndModify/
+     */
+    collation: Optional<Document>;
+
+    /**
      * The maximum amount of time to allow the query to run.
      *
-     * @see http://docs.mongodb.org/manual/reference/command/findAndModify/
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     *     
+     * @see https://docs.mongodb.org/manual/reference/command/findAndModify/
      */ 
     maxTimeMS: Optional<Int64>;
 
     /** 
      * Limits the fields to return for all matching documents.
      *
-     * @see http://docs.mongodb.org/manual/tutorial/project-fields-from-query-results
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     *
+     * Note: this option is mapped to the "fields" findAndModify field.
```

Comment:
Would it make more sense to say "findAndModify command option" instead of "findAndModify field"?